sample/sample.rb

```diff
@@ -0,0 +1,21 @@
+require 'clever-api'
+
+# Setup authorization
+CleverAPI.configure do |config|
+  # Configure OAuth2 access token for authorization: oauth
+  config.access_token = 'DEMO_TOKEN'
+end
+
+api_instance = CleverAPI::DefaultApi.new
```

Comment:
After discussion with @meliaj going to go with `Clever::DataApi` and `Clever::EventsApi`